src/Domain/Value/QueryParameter/QueryParameter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of storyblok/php-content-api-client.
+ *
+ * (c) Storyblok GmbH <info@storyblok.com>
+ * in cooperation with SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Api\Domain\Value\QueryParameter;
+
+/**
+ * Represents top level query parameters which are holding simple string values.
+ *
+ * @author Frank Stelzer <dev@frankstelzer.de>
+ */
+interface QueryParameter
```

Comment:
I think we don't need this interface. Because `getName` for instance is not used anywhere.

Reply:
Additionally its nothing users can use to implement their own query parameters. so there is no point of keeping it.

Reply:
Any suggestion where to store the date format then? I think we should hardcode the same date format in six different value object. But I didn't found any matching alternative in the existing code.